obkvrpc/connection_pool.go

```diff
@@ -68,35 +76,41 @@ func NewConnectionPool(option *PoolOption) (*ConnectionPool, error) {
 		rwMutexes:   make([]sync.RWMutex, 0, option.connPoolMaxConnSize),
 	}
 
-	connectionOption := NewConnectionOption(pool.option.ip, pool.option.port, pool.option.connectTimeout, pool.option.loginTimeout,
-		pool.option.tenantName, pool.option.databaseName, pool.option.userName, pool.option.password)
+	if option.maxConnectionAge > 0 || option.enableSLBLoadBalance {
```

Comment:
why || ?

Reply:
maxConnectionAge and enableSLBLoadBalance are two separate option, if maxConnectionAge is set, we will init connLifeCycleMgr inside connection mgr and start connMgr, which will monitor the life cycle of each connection. if enableSLBLoadBalance is set, we also init SLBLoader inside connMgr and start connMgr to monitor the change of SLB list.